caps/types.go

```diff
@@ -31,6 +31,9 @@ type Type interface {
 	Name() string
 	// Sanitize a capability (altering if necessary).
 	Sanitize(c *Capability) error
+	// Obtain the security snippet for the given security system.
+	// If no security snippet is needed, hand out empty string.
```

Comment:
// SecuritySnippet returns the configuration snippet that should be used by
    // the given security system to enable this capability to be consumed.
    // An empty snippet is returned when the capability doesn't require anything
    // from the security system to work, in addition to the default configuration.
    // ErrUnknownSecurity is returned when the capability cannot deal with the
    // requested security system.